api/authFunctions.test.tsx

```diff
@@ -0,0 +1,18 @@
+test('get weekly picks mock function', async () => {
+    const users = { userId: '66174f2362ec891167be' };
+    const resp = { data: users };
+
+    // Mocking the getWeeklyPicks function
+    jest.mock("./authFunctions", () => ({
+        getWeeklyPicks: jest.fn().mockResolvedValue(resp)
+    }));
+
+    // Importing the mocked function
+    const { getWeeklyPicks: mockGetWeeklyPicks } = require("./authFunctions");
```

Comment:
We still need to call authFunctions and use the actual getWeeekyPicks(). Let's keep this for now and revisit after MVP.